test/controllers/github_events_controller_test.rb

```diff
@@ -15,7 +15,7 @@ class GithubEventsControllerTest < ActionController::TestCase
 
     request.headers['HTTP_X_HUB_SIGNATURE'] = signature
 
-    post :payload, params: payload, session: { format: :json, number: 1, pull_request: { state: 'open' }, repository: { full_name: repository.name } }
+    post :payload, body: payload, params: { format: :json, number: 1, pull_request: { state: 'open' }, repository: { full_name: repository.name } }
```

Comment:
It seems like 1231564e8ac8711a7987fcba49f1005361443082 and this commit might be able to be combined, what was your reasoning for keeping them separate?

Reply:
No reason, I was just committing as I made progress.  I can squash.

Reply:
Sure, I think it would be a good idea to squash these, but it's really minor.

Reply:
Fixed: 1297d8bfdd257f40df2d36be3d6fccad31873bb4